src/main/java/org/broadinstitute/hellbender/engine/ReadWalker.java

```diff
@@ -72,9 +73,11 @@ public void traverse() {
         final CountingReadFilter countedFilter = disable_all_read_filters ?
                                                     new CountingReadFilter("Allow all", ReadFilterLibrary.ALLOW_ALL_READS ) :
                                                     makeReadFilter();
+        final ReadTransformer transformer = makeReadTransformer();
 
         StreamSupport.stream(reads.spliterator(), false)
                 .filter(countedFilter)
+                .map(transformer)
```

Comment:
Keep it as now (filter/transformer). I guess that that's why I change it here, because probably the tests were failing.